Userland/Libraries/LibWeb/HTML/CustomElements/CustomElementRegistry.cpp

```diff
@@ -337,6 +337,21 @@ Variant<JS::Handle<WebIDL::CallbackType>, JS::Value> CustomElementRegistry::get(
     return JS::js_undefined();
 }
 
+// https://html.spec.whatwg.org/multipage/custom-elements.html#dom-customelementregistry-get
```

Comment:
I think this link should be: https://html.spec.whatwg.org/multipage/custom-elements.html#dom-customelementregistry-getname